### ISSUE TITLE
UV4 forces two page view?

### ISSUE THREAD

**Issue by nicolasfranck** [UniversalViewer/universalviewer#872]
**UV version:**

```
 universalviewer@v4
```

**I'm submitting a:**

- [x] bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/V4-Examples) with a repro of your issue and include a link to it below
- [ ] feature request => please use the [user stories repo](https://github.com/UniversalViewer/user-stories)
- [ ] support request => Please do not submit support requests here, use [stackoverflow](http://stackoverflow.com/questions/tagged/iiif)

**Current behavior:**

When I supply a version 2 manifest WITHOUT
`viewingHint` set to `paged`, then still the UV
shows the pages side by side.

e.g. https://adore.ugent.be/IIIF/manifests/archive.ugent.be:018970A2-B1E8-11DF-A2E0-A70579F64438

**Comment by glenrobson** [comment]
We found the same issue in the IIIF training when the `viewingHint` was set to `individuals`. It still shows two up view. See the following links:

 * [Manifest](https://iiif-test.github.io/test3/manifests/4_page_manifest.json)
 * [UV4](https://uv-v4.netlify.app/#?manifest=https://iiif-test.github.io/test3/manifests/4_page_manifest.json&c=&m=&cv=&xywh=0%2C-200%2C4031%2C3423)
 * [UV3](https://uv-v3.netlify.app/#?c=&m=&s=&cv=&manifest=https://iiif-test.github.io/test3/manifests/4_page_manifest.json) - which works correctly for `individuals`

**Comment by edsilv** [comment]
Fixed: https://www.universalviewer.dev/#?xywh=1314%2C907%2C13867%2C9280&iiifManifestId=https%3A%2F%2Fadore.ugent.be%2FIIIF%2Fmanifests%2Farchive.ugent.be%3A018970A2-B1E8-11DF-A2E0-A70579F64438&cv=

**Comment by nicolasfranck** [comment]
Ok, I've tried it with a non-paged canvas, trying to do the following:

* front-page
* two-page-foldout -> non-paged
* left1 + right1
* left2 + right2

but I got

* front-page
* two-page-foldout
* left1
* right1 + left2

e.g. 

It seems like that AFTER a non-paged canvas, everything restarts, with a single canvas included (as with the front-page).
As I would not expect that single canvas after that, I can imagine other use cases. Is this a hole in the specification?
